.github/workflows/cibuild.yml

```diff
@@ -115,4 +115,15 @@ jobs:
           distribution: temurin
           cache: maven
       - name: Build container image with Jib, push to Dockerhub
-        run: mvn --batch-mode compile com.google.cloud.tools:jib-maven-plugin:build -Djib.to.tags=latest,${{ github.sha }}
+        run: |
+          # we give the container three tags
+          #   - latest
+          #   - the git SHA1
+          #   - a string like "2.3-2022-12-12-21_38"
```

Comment:
I started reading this like the first part would be some date but didn't take too long to figure out what was really happening, although I also had the help of the code here to figure it out. I don't know if changing it to be `v2.3-2022-12-12-21_38` would be clearer, another problem is that the `-` is used as a separator between the version and the datetime but also within the datetime. Not sure if we could use some different separator.

Reply:
How would you like

```
v2.3-2022-12-12T21_38
```

which uses `T` like ISO8601 an im happing putting a `v` there to make it easier.

Does HSL uses the containers, btw?

Reply:
That looks fine and no, HSL does not use them at least yet so I think you have more authority to decide what is suitable for you, because you actually use the images.

Reply:
Done.